paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1309,6 +1309,28 @@ def _run_instance_worker(cluster_data, instances_out, green_light):
                 cluster_data.instances_queue.task_done()
                 instances_out.put(instance_config)
                 continue
+
+            # `is not None` is necessary here because I'm adding the new attribute to the API response at the same time as I'm adding this reference.
+            if long_running_status.active_shas is not None:
```

Comment:
due to the way generated code works atm, you'll need to do this:

    try:
        active_shas = long_running_status.active_shas
    except paasta_tools.paastaapi.ApiAttributeException:
        active_shas = None

i'm getting this fixed, already got PR merged into openapi generator, just didn't switch to new generator image in Makefile

Reply:
Interesting. When I test my branch, I get None. I'll add this snippet in to be cautious.